help/help.adoc

```diff
@@ -264,129 +324,77 @@ This procedure allows users to delete a Virtual Network using Azure Services.
 |JobStep Timeout |Timeout for the step execution in minutes. Blank means no timeout.
 |===
 
-image::cloudbees-common::cd-plugins/ec-azure/parameters/delete-vnet.png[image]
+image::cloudbees-common-sda::cd-plugins/ec-azure/parameters/delete-vnet.png[image]
 
-[[DeleteDatabase]]
-
-
-[[DeleteDatabase]]
-=== DeleteDatabase
-
-This procedure allows users to delete a database on a database server using Azure Services.
-
-[cols=",",options="header",]
-|===
-|Parameter |Description
-|Configuration |The name of the configuration which holds all the connection information for Windows Azure. (Required)
-|Resource Group Name |The name of the Resource Group to which the Azure SQL Database Server belongs. (Required)
-|Server name |The name of the Azure SQL Database Server on which the database is hosted. (Required)
-|Database name |The name of the Azure SQL Database to be operated on (Updated or created). (Required)
-|JobStep Timeout |Timeout for the step execution in minutes. Blank means no timeout.
-|===
-
-image::cloudbees-common::cd-plugins/ec-azure/parameters/delete-database.png[image]
-
-[[Delete-NetworkSecurityGroup]]
-
-
-[[Delete_NetworkSecurityGroup]]
-=== Delete NetworkSecurityGroup
+[[CreateVM]]
+=== CreateVM
 
-This procedure allows users to delete a Network Security Group.
+This procedure allows users to create a VM using Azure Services.
 
-[cols=",",options="header",]
+[cols="1,1",options="header",]
 |===
 |Parameter |Description
 |Configuration |The name of the configuration which holds all the connection information for Windows Azure. (Required)
-|Network Security Group |Name of the Network security Group which is to be deleted. (Required)
-|Resource Group Name |The name of the resource group in which network security group is present. (Required)
-|JobStep Timeout |Timeout for the step execution in minutes. Blank means no timeout.
-|===
-
-image::cloudbees-common::cd-plugins/ec-azure/parameters/delete-network-security-group.png[image]
-
-[[Delete-NetworkSecurityRule]]
+|Server Name |Name of the new Server. (Required)
+|Number of Instances |Number of instances to be created. (Required)
+|Image URN a|The image URN in the form publisherName:offer:skus:version/ The source VHD link of a user image. For example, `\https://foo.blob.core.windows.net/system/Microsoft.Compute/Images/vhds/snap-bar.vhd`.(Required)
 
+====
+NOTE: For your user image to be accepted, it must belong to a storage account in same resource group as VM itself (this is Azure's requirement).
+====
```

Comment:
We can omit the `====` before and after the note admonition, so it doesn't include the bold outline around the note box.
```suggestion
NOTE: For your user image to be accepted, it must belong to a storage account in same resource group as VM itself (this is Azure's requirement).
```

Reply:
updated